### ISSUE TITLE
Bold claims in the readme

### ISSUE THREAD

**Issue by ZeChArtiahSaher** [ms-jpq/chadtree#352]
So I before trying out I read the readme of the project it has these bold claims like how awesome it is , how it's 21st century how superior it is. Ngl I'm gonna say boldly - it's not that much better than NERD tree, here's why. Interface.

Chad tree fails to improve on some limitations of the interface that NERD tree has, in some ways it goes backwards by not respecting anything that NERD tree actually had working somewhat alright:
- chad keys are not as intuitive as nerd tree
- doesn't beat a simple '?' key in any way. In nerdtree window as a quick help 
- in neovide horizontal mouse scrolling on osx is utter trash and somehow there is no clear way to unset it unless I waste my evening diggin the docs

Honestly the one thing that you can make a better nerd tree besides all the algo improvements is actually UI/UX. And UI/UX in a sense if there was ever ability to drag files/folders that would be something different for example. That would be worth yet another vscode tree type plugin. Everything else is a sugar coating.

tl;dr; chadtree may be faster, but excels at making the user interface even worse than that of NERD's tree...

**Comment by clarity20** [comment]
I second the suggestion for a '?' feature (although I prefer to press <F1>). This plugin has some nice ideas, I want to like it but the key mappings are not very discoverable.

**Comment by clarity20** [comment]
I second the suggestion of a '?' feature (although I prefer to press F1). It would be nice 
if the key mappings were more discoverable.
